frontend/src/components/RoutePlanner.tsx

```diff
@@ -55,18 +56,52 @@ const RoutePlanner: React.FC = () => {
           prefixDataTestId="destination-input"
         />
       </IonItem>
-      <IonButton type="submit" size="default" expand="block" onClick={submitInput}>Search routes</IonButton>
+      <IonButton type="submit" size="default" expand="block">Search routes</IonButton>
```

Comment:
Maybe that's not directly related to this issue, but changing the inputs for date/time is not triggering a new query (as origin and destination are doing). As I understand the "search routes" button is not doing anything (so no help), right?

Edit: Just saw that date/time input is not connected to anything query-related yet (on frontend). So that seems to be something for another issue indeed.

Reply:
> Edit: Just saw that date/time input is not connected to anything query-related yet (on frontend). So that seems to be something for another issue indeed.

Jup that is still missing #278